Challenge_1/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿//Challenge 1
+Console.Write("Gebe Ergebnis ein: ");
+double ergebnis = Convert.ToDouble(Console.ReadLine());
```

Comment:
Schau dir mal diese Funktion an: https://learn.microsoft.com/de-de/dotnet/api/system.double.tryparse?view=net-8.0
damit kannst du prüfen, ob die Eingabe auch in deinen double konvertiert werden kann. Dadurch kannst du Laufzeitfehler verhindern.